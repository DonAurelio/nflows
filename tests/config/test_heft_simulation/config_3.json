{
    "dag_file": "./tests/workflows/test_heft_simulation/config_3.dot",
    "scheduler_type": "heft",
    "mapper_type": "simulation",
    "mapper_mem_policy_type": "default",

    "core_avail_mask": "0x01000001",
    "flops_per_cycle": 1000000,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1,

    "distance_matrices": {
        "latency": "./tests/system/test_heft_simulation/3_lat.txt",
        "bandwidth": "./tests/system/test_heft_simulation/3_bw.txt"
    },

    "log_base_name": "./tests/output/test_heft_simulation/config_3",
    "log_date_format": ""
}
