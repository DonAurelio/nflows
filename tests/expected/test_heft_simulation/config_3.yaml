runtime:
  core_availability:
    0: {avail_until: 440}
    24: {avail_until: 180}

trace:
  exec_name_total_offsets:
    Task_3: {start: 340, end: 440, payload: 80}
    Task_1: {start: 0, end: 180, payload: 160}
    Task_2: {start: 0, end: 340, payload: 320}
  